pkg/metrics/init.go

```diff
@@ -129,4 +131,24 @@ func GetQueueMetrics(name string) CoreQueueMetrics {
 		m.queues[name] = queueMetrics
 		return queueMetrics
 	}
+}
+
```

Comment:
Can we add some comments here, saying this is according to https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels.

Reply:
Done